build/buildspec.yaml

```diff
@@ -0,0 +1,21 @@
+---
+version: 0.2
+
+env:
+  parameter-store:
+    PYPI_TOKEN: /prod/futures/polo-futures-sdk-python/pypi_token
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.8
+    commands:
+      - pip install --upgrade pip
+      - pip install --upgrade setuptools wheel
+      - pip install twine
+
+  build:
+    commands:
+      - python setup.py sdist bdist_wheel
+      - echo "[pypi]\nusername = __token__\npassword = $PYPI_TOKEN" > ~/.pypirc
```

Comment:
https://packaging.python.org/specifications/pypirc/ recommends using https://pypi.org/project/keyring/ to securely retrieve the `password` instead of storing this in the `~/.pypirc` file